CanvasCore/CanvasCore/React Native Modules/CanvasCrashlytics.m

```diff
@@ -96,42 +96,52 @@ - (dispatch_queue_t)methodQueue
         domain = DefaultDomain;
 
     NSError *error2 = [NSError errorWithDomain:domain code:code userInfo:error];
-    [[Crashlytics sharedInstance] recordError:error2];
+    [FIRCrashlytics.crashlytics recordError:error2];
 }
 
 RCT_EXPORT_METHOD(crash)
 {
-    [[Crashlytics sharedInstance] crash];
+    assert(NO);
 }
 
 RCT_EXPORT_METHOD(throwException)
 {
-    [[Crashlytics sharedInstance] throwException];
+    assert(NO);
 }
 
 RCT_EXPORT_METHOD(setUserIdentifier:(NSString *)userIdentifier)
 {
-    [[Crashlytics sharedInstance] setUserIdentifier:userIdentifier];
+    [[FIRCrashlytics crashlytics] setUserID:userIdentifier];
 }
 
 RCT_EXPORT_METHOD(setUserName:(NSString *)userName)
 {
-    [[Crashlytics sharedInstance] setUserName:userName];
+    //      this has been deprecated in firebase and discouraged
+    //      https://firebase.google.com/docs/crashlytics/upgrade-sdk?authuser=1#switch-pods
+    /*
+     We adopted the method name setUserID to be consistent with other Firebase APIs and removed setUserName and setUserEmail to discourage logging PII through Crashlytics.
+     */
+    //[[Crashlytics sharedInstance] setUserName:userName];
 }
 
 RCT_EXPORT_METHOD(setUserEmail:(NSString *)email)
 {
-    [[Crashlytics sharedInstance] setUserEmail:email];
+    //      this has been deprecated in firebase and discouraged
+    //      https://firebase.google.com/docs/crashlytics/upgrade-sdk?authuser=1#switch-pods
+    /*
+     We adopted the method name setUserID to be consistent with other Firebase APIs and removed setUserName and setUserEmail to discourage logging PII through Crashlytics.
+     */
+//    [[Crashlytics sharedInstance] setUserEmail:email];
 }
```

Comment:
Can we just remove these two methods on the objc side, and also remove them on the js side (src/common/CanvasCrashlytics.js)? As far as I can tell these aren't used anyway.